edge/pkg/edged/edged.go

```diff
@@ -913,13 +913,17 @@ func (e *edged) consumePodAddition(namespacedName *types.NamespacedName) error {
 	}
 
 	podUID := pod.GetUID()
-	curPodStatus, err := e.podCache.GetNewerThan(podUID, e.podLastSyncTime[podUID])
+	t, ok := e.podLastSyncTime.Load(podUID)
+	if !ok {
+		t = new(time.Time)
```

Comment:
t should not be the pointer, t = time.time{}

Reply:
> t should not be the pointer, t = time.time{}

Got it.